scripts/install-operator-sdk.sh

```diff
@@ -14,21 +14,29 @@ set -eo pipefail
 
 SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ROOT_DIR="$SCRIPTS_DIR/.."
-DEFAULT_OPERATOR_SDK_VERSION="1.7.1"
+DEFAULT_OPERATOR_SDK_BIN_PATH="$ROOT_DIR/../../aws-controllers-k8s/code-generator/bin"
+OPERATOR_SDK_BIN_PATH=${OPERATOR_SDK_BIN_PATH:-$DEFAULT_OPERATOR_SDK_BIN_PATH}
+DEFAULT_OPERATOR_SDK_VERSION="1.17.0"
 
 source "${SCRIPTS_DIR}/lib/common.sh"
 
 __operator_sdk_version="${1}"
 if [ "x${__operator_sdk_version}" == "x" ]; then
     __operator_sdk_version=${OPERATOR_SDK_VERSION:-$DEFAULT_OPERATOR_SDK_VERSION}
 fi
-if ! is_installed operator-sdk; then
+if ! is_installed ${OPERATOR_SDK_BIN_PATH}; then
```

Comment:
Similarly to @RedbackThomson comments this should be `${OPERATOR_SDK_BIN_PATH}/operator-sdk`

Reply:
Fixed.